### PR TITLE
6, 8 - Answers and questions models

### PR DESCRIPTION
**Pull request by gabrielestes**
1. Generated answers model
2. Generated questions model
3. Modified each model above to include ActiveRecord relationships as designed in the database map. Images below.
4. Generated and ran migrations.

screen shot 2017-12-02 at 5 54 25 pm

screen shot 2017-12-02 at 6 22 24 pm

### CODE REVIEW

**Comment by gabrielestes** [comment]
[Add answers model](https://trello.com/c/cVcdqNg7/6-add-answers-model)

**Comment by gabrielestes** [comment]
[Add questions model](https://trello.com/c/yiXADApP/8-add-questions-model)

File: app/models/answer.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Answer < ApplicationRecord
+  belongs_to :question,
+  belongs_to :contact,
```
Comment (TurnerMooreDavis) [inline app/models/answer.rb]: should this maybe belong to profile instead?  we can always trace where the answer came from by looking at the questions virtual form

File: db/schema.rb [hunk 2]
```diff
@@ -54,4 +66,19 @@
...
 
+  create_table "questions", force: :cascade do |t|
+    t.text "body"
+    t.bigint "virtualform_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["virtualform_id"], name: "index_questions_on_virtualform_id"
+  end
+
+  create_table "virtualforms", force: :cascade do |t|
```
Comment (TurnerMooreDavis) [inline db/schema.rb:77]: why is this here?  I thought we created this table in a separate pull request?

File: app/models/answer.rb (hunk 1 above)
Comment (gabrielestes) [reply app/models/answer.rb]: That's a good point. I'll make the change.

File: db/schema.rb (hunk 2 above)
Comment (gabrielestes) [reply db/schema.rb]: I included it because it was more convenient for the migrations to include `virtualform_id` in the original. That way, I wouldn't have to create a new migration to add `virtualform_id`. It would've been more organized to have a separate PR, and I didn't document that in the PR. My bad!

Comment (gabrielestes) [reply db/schema.rb:77]: Hm that's odd. Not sure how that happened. 🤔  I'll hold off on merging this until I figure it out.

Comment (gabrielestes) [reply db/schema.rb:77]: @TurnerMooreDavis I originally rebased this branch off of the Virtualform branch, but GitHub is not recognizing the similarity between the two. Not sure why.

Comment (gabrielestes) [reply db/schema.rb:77]: @TurnerMooreDavis I think it's fine since it's just updating the schema to a state it's already in 🤔

**Review by SmartGameSystems — COMMENTED** [review]
Approving on behalf of Turner

**Review by SmartGameSystems — APPROVED** [review]
